lib/tasks/update_cms_tags.rake

```diff
@@ -11,6 +11,8 @@ namespace :cms_update_2 do
       layout.content = layout.content.gsub(/\{\{ ?cms:page:([\w]+):rich_text ?\}\}/, '{{ cms:wysiwyg \1 }}') if layout.content.is_a? String
       layout.content = layout.content.gsub(/\{\{ ?cms:page:([\w]+):([^:]*) ?\}\}/, '{{ cms:\2 \1 }}') if layout.content.is_a? String
       layout.content = layout.content.gsub(/\{\{ ?cms:field:([\w]+):string ?\}\}/, '{{ cms:text \1, render: false }}') if layout.content.is_a? String
+      # convert boolean to checkbox
+      layout.content = layout.content.gsub(/\{\{ ?cms:field:([\w]+):boolean ?\}\}/, '{{ cms:checkbox \1, render: false }}') if layout.content.is_a? String
```

Comment:
Looks like this isn't working for me, I still got the same error even after relaunching the rake task.
So I have tried to rollback the two migrations and it breaks on this one:

== 20200103163139 RenameComfyCmsBlocksToComfyCmsFragments: reverting ==========
rake aborted!
StandardError: An error has occurred, this and all later migrations canceled:

This migration uses change_column, which is not automatically reversible.
To make the migration reversible you can either:
1. Define #up and #down methods in place of the #change method.
2. Use the #reversible method to define reversible behavior.

Reply:
Uhm, that's weird.
The migrations are not reversible at the moment; I'm literally working on this and checking if it's possible at all.

I'll have another look.
Anyway, you can check and change it manually yourself and see what happens (perhaps also understanding why :) ).

In the layout section of the CMS, there's a layout called `Versions on the left, article on the right`. If you try to edit this layout you should find the boolean tag which isn't working.
It worked for me by replacing `boolean` with checkbox.

Reply:
I think I know what is happening here:
This is the original value `cms:field:display_version_alert:boolean`, while now I have on my db `{{ cms:boolean  display_version_alert, render: false }}`, changed after being caught by this other regexp `layout.content = layout.content.gsub(/\{\{ ?cms:field:([\w]+):([^:]*) ?\}\}/, '{{ cms:\2 \1, render: false }}')` before you introduced this new one.
This will not happen when running the script on prod.

Reply:
Ah, sure! Forgot about this completely.
Yeah if you bring the boolean tag back to the original and relaunch the update script again it should work.
Sorry and thanks for checking!